taskcluster/ci/beetmover/kind.yml

```diff
@@ -26,3 +26,11 @@ job-template:
         artifact_map: taskcluster/fenix_taskgraph/manifests/fenix_candidates.yml
     treeherder:
         job-symbol: BM
+    bucket-scope:
+        by-level:
+            '3':
+                by-build-type:
+                    'nightly': "project:mobile:fenix:releng:beetmover:bucket:nightly"
+                    (release|beta): "project:mobile:fenix:releng:beetmover:bucket:release"
+                    default: "project:mobile:fenix:releng:beetmover:bucket:dep"
+            default: "project:mobile:fenix:releng:beetmover:bucket:dep"
```

Comment:
Perfect. Have you run taskgraph-diff to make sure the scopes look right in each graph?